components/homekit/esp_hap_platform/Kconfig

```diff
@@ -85,3 +85,19 @@ menu "HAP HTTP Server"
             Set the Maximum number of URI handlers that the HTTP Server should allow.
 
 endmenu
+
+menu "HAP Platform"
+
+    config HAP_PLATFORM_DEF_NVS_RUNTIME_PARTITION
+        string "HAP Platform runtime NVS partition name"
+        default "nvs"
+        help
+            Set the runtime NVS partition name for HomeKit use
+
+    config HAP_PLATFORM_DEF_NVS_FACTORY_PARTITION
+        string "HAP platform factory NVS partition name"
```

Comment:
This can be rephrased to just `Factory NVS partition name` since HAP Platform as the the prefix would be a bit redundant here as it is already mentioned in menu.